README.md

```diff
@@ -67,4 +67,9 @@ $ cp .env.oss .env
 
         $ yarn sync-schema
 
-* There are some suggested VSCode extensions in `.vscode/extensions.json` and additional docs at [docs/vscode.md](docs/vscode.md).
+* [Circle CI](https://circleci.com/docs/1.0/npm-continuous-deployment/) is set up to publish reaction-force in consistent and reliable way. To update the package version and deploy it on npm. You can read more about publishing and updating npm packages [here](https://docs.npmjs.com/getting-started/publishing-npm-packages):
```

Comment:
Maybe it would also be more helpful to cut to the chase initially, so after "the following commands ... deploy it to NPM" list the commands. Then in the next paragraph explain what those commands do.